ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -707,6 +707,27 @@ func (ipfs *Connector) BlockGet(ctx context.Context, c cid.Cid) ([]byte, error)
 	return ipfs.postCtx(ctx, url, "", nil)
 }
 
+// FetchRefs asks IPFS to download blocks recursively to the given depth.
+// It discards the response, but waits until it completes.
+func (ipfs *Connector) FetchRefs(ctx context.Context, c cid.Cid, maxDepth int) error {
```

Comment:
Maybe remove

Reply:
leaving commented. We will use it one day for something. No sense to have to re-implement.